common.gypi

```diff
@@ -31,7 +31,7 @@
         'os_posix': 1,
         'v8_postmortem_support%': 'true',
       }],
-      ['GENERATOR == "ninja" or OS== "mac"', {
+      ['GENERATOR == "ninja" or OS=="mac"', {
```

Comment:
Judging by the other condition on the same line, it looks like a space should be added on the left hand side instead of removing the one space on the right hand side?